django_project/blog/templates/blog/post/post_detail.html

```diff
@@ -61,13 +61,13 @@ <h1 class="sml-margin-top-bottom">
 <div class="social-share-container">
   <ul class="social-share">
     <li>
-      <a class="resp-sharing-button__link twitter"
+      <a class="resp-sharing-button__link x"
```

Comment:
Use x.com link

Reply:
alright

Reply:
@freedompraise can you change the logo to white using the SVG I uploaded?

Reply:
Going throuh this now, I'm a bit confused; how does the ngrok comand work? Cos it's not a package on Linux

Reply:
Have you tried reading the ngrok website?
https://ngrok.com/download

All do do is ngrok `ngrok http 80` and it will create a tunnel to your computer so that it can be accessed from any computer in the world (It basically makes your computer a web server).

Then, you'll be able to test a share link on X.com because the link will not be localhost, but an actual URL that is resolvable.

Reply:
ngrok closes on my PC after I run the command. Is it supposed to work that way?

Reply:
I'll try this out next week. I am traveling this weekend.